### ISSUE TITLE
When using "textOverflow: 'none'," as style for y axis labels, the wrapped labels are getting cropped.

### ISSUE THREAD

**Issue by praveen-kaple** [highcharts/highcharts#10761]
#### Expected behaviour
When using "textOverflow: 'none'," as style for y axis labels, the wrapped labels should not be cropped.

#### Actual behaviour
When using "textOverflow: 'none'," as style for y axis labels, the wrapped labels are getting cropped.

#### Live demo with steps to reproduce
https://jsfiddle.net/wg4q20rb/3/

#### Product version
6.0.3

#### Affected browser(s)
Chrome, FF

**Comment by pawelfus** [comment]
Hi @praveen-kaple 

Thanks for reporting the issue. This is caused by very long labels that don't have enough space to render. By default, if you don't set [`chart.marginLeft`](https://api.highcharts.com/highcharts/chart.marginLeft), we dynamically calculate left margin, up to 1/3 width of the chart. It's a maximum width for long labels without word-wrapping. If labels are longer, we render them in multiple lines. In your case, labels are so long that even word-wrapping is not enough.

In short, you can  either set `chart.marginBottom` so cropped label will be visible (demo: https://jsfiddle.net/3r0mv4f7/1/) or increase `chart.marginLeft` (demo: https://jsfiddle.net/3r0mv4f7/2/) - so labels are rendered in two lines only.
